hello-pepr-reconcile/capabilities/reconcile.e2e.test.ts

```diff
@@ -0,0 +1,36 @@
+import { beforeAll, afterAll, describe, it, jest, expect } from "@jest/globals";
+import { TestRunCfg } from "helpers/src/TestRunCfg";
+import { secs, mins, sleep, timed } from "helpers/src/time";
+import { K8s, kind } from "kubernetes-fluent-client";
+import { fullCreate } from "helpers/src/general";
+import { moduleUp, moduleDown, untilLogged, logs } from "helpers/src/pepr";
+const trc = new TestRunCfg(__filename);
+
+// KUBECONFIG=$(k3d kubeconfig write pexex-hello-pepr-reconcile-e2e) k9s
+describe("reconcile.ts", () => {
+  beforeAll(async () => await moduleUp({ verbose: true }), mins(2));
+  afterAll(async () => await moduleDown(), mins(2))
+
+  const LOG_RESULTS: string[] = ["three", "two", "one"];
```

Comment:
```suggestion
```
It's a good practice to keep test data as "local" to the tests that need it as you can (i.e. in the same block, if possible) -- recommend not defining this here & just in-lining it into the test that needs it (as I did in the suggestion below).

Reply:
addressed